tests/unit/adapter/test_maintenance_db.py

```diff
@@ -15,65 +15,81 @@ def teardown_module():
 
 
 def setup_function():
-    DB_CLIENT.jobdb.drop_collection("maintenance")
+    DB_CLIENT.maintenance_db.drop_collection("maintenance")
```

Comment:
In the adapter implementation we use db.collection = jobDB.maintenance collection 🤔 That should ideally be reflected here too in order to not be confusing

Reply:
That is true, I'll fix